server/requirements.txt

```diff
@@ -19,5 +19,6 @@ pyesbulk>=2.0.1
 PyJwt
 psycopg2
 requests
+selinux
```

Comment:
It would be cool if you would make the commensurate changes to `jenkins/dockerfile` and `server/rpm/pbench-server.spec.j2` if there is an RPM for this package.